### PR TITLE
Ben/features/test

### PR DESCRIPTION
**Pull request by benSmith1981**
This is broken code dont approve it

### CODE REVIEW

File: OMDBMovies/AppDelegate.swift [hunk 1]
```diff
@@ -13,11 +13,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
```
Comment (benSmith1981) [inline OMDBMovies/AppDelegate.swift:16]: Hey why did you remove the main app method!

**Review by benSmith1981 — COMMENTED** [review]
terrible

**Comment by frankfrisco** [comment]
let's brake more code :)

**Comment by Ewoud1988** [comment]
According to 'Le Ties', you're a natural. Way the go man!
